bin/interpreter.sh

```diff
@@ -86,6 +89,9 @@ ZEPPELIN_SERVER=org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer
 INTERPRETER_ID=$(basename "${INTERPRETER_DIR}")
 ZEPPELIN_PID="${ZEPPELIN_PID_DIR}/zeppelin-interpreter-${INTERPRETER_ID}-${ZEPPELIN_IDENT_STRING}-${HOSTNAME}.pid"
 ZEPPELIN_LOGFILE="${ZEPPELIN_LOG_DIR}/zeppelin-interpreter-"
+if [[ ! -z "$INTERPRETER_NAME" ]]; then
+    ZEPPELIN_LOGFILE+="${INTERPRETER_NAME}-"
+fi
```

Comment:
Yes, even I initially thought about removing `INTERPRETER_ID` but then no harm in it, and I thought about the case where the user would want to `tail -f *jdbc*` and all rest of the interpreters as HIVE, PHOENIX or POSTGRES, etc.